packages/ui/layout/BottomNavBar/NavBarButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Image } from '@chakra-ui/react';
+
+export type NavBarButtonProps = {
+  onClick: VoidFunction;
+  selected: boolean;
+  buttonImageSrc: [string, string];
+  buttonText: string;
+};
+
+export default function NavBarButton({
+  onClick,
+  selected,
+  buttonImageSrc,
+  buttonText,
+}: NavBarButtonProps) {
+  const [unselectedButton, selectedButton] = buttonImageSrc;
+
+  return (
+    <Box
+      color={selected ? 'gray.600' : 'gray.500'}
+      fontSize="9px"
+      alignItems="center"
+      justifyContent="center"
+      onClick={onClick}
+      as="button"
+    >
```

Comment:
```suggestion
    <Flex
      color={selected ? 'gray.600' : 'gray.500'}
      fontSize="9px"
      align="center"
      justify="center"
      onClick={onClick}
      as="button"
    >
```

`Flex`로 바꿔도 괜찮을 것 같아요!!

Reply:
`Flex` 로 바꾸면 direction 이 `row` 가 기본으로 되어 있어서, `column` 방향으로 쌓이도록 `Box` 를 사용했는데, 혹시 `Flex` 를 사용해야하는 이유가 있을까요?
<img width="277" alt="image" src="https://github.com/Anifriends/Anifriends-Frontend/assets/74397358/dbaf2e19-41a5-4dfb-a284-308ad45cbc49">

Reply:
아 정정합니다. `justifyContent`, `alignItems`를 없애도 무방할 것 같아요!!! 🙏🙋🏻‍♂️